### PR TITLE
[ELK] Update license info

### PR DESCRIPTION
**Pull request by valeriocos**
This code updates license information for every source code file in ELK.

### CODE REVIEW

File: utils/e2k.py [hunk 1]
```diff
@@ -58,6 +55,7 @@ def get_params():
 
 
 if __name__ == '__main__':
+    """Enriched to Kibana Publisher"""
```
Comment (sanacl) [inline utils/e2k.py]: This comment should not be part of the commit.

File: utils/gelk.py [hunk 2]
```diff
@@ -56,6 +53,7 @@ def get_connector_from_name(name, connectors):
 
 
 if __name__ == '__main__':
+    """Gelk: perceval2ocean and ocean2kibana"""
```
Comment (sanacl) [inline utils/gelk.py]: This comment should not be part of the comment

File: utils/gelk_mapping.py [hunk 3]
```diff
@@ -131,6 +128,7 @@ def find_ds_mapping(data_source, es_major_version):
 
 
 if __name__ == '__main__':
+    """Tool to show the mapping used in a GrimoireELK data source"""
```
Comment (sanacl) [inline utils/gelk_mapping.py]: This comment should not be part of the commit.

File: utils/get_git_repos.py [hunk 4]
```diff
@@ -137,6 +134,8 @@ def get_repositories(owner, token, nrepos):
 
 
 if __name__ == '__main__':
+    """GitHub repository listing for an org"""
```
Comment (sanacl) [inline utils/get_git_repos.py]: This comment should not be part of the commit.

File: utils/gh2arthur.py [hunk 5]
```diff
@@ -226,6 +223,7 @@ def insert_projects_mapping(db_projects_map, project, repositories):
 
 
 if __name__ == '__main__':
+    """GitHub to Kibana"""
```
Comment (sanacl) [inline utils/gh2arthur.py]: This comment should not be part of the commit.

File: utils/gh2k.py [hunk 6]
```diff
@@ -287,6 +284,7 @@ def get_oauth():
 
 
 if __name__ == '__main__':
+    """GitHub to Kibana"""
```
Comment (sanacl) [inline utils/gh2k.py]: This comment should not be part of the commit.

File: utils/index_mapping.py [hunk 7]
```diff
@@ -350,6 +347,7 @@ def export_items(elastic_url, in_index, out_index, elastic_url_out=None,
 
 
 if __name__ == '__main__':
+    """Tool to ensure correct mappings in GrimoireLab indexes"""
```
Comment (sanacl) [inline utils/index_mapping.py]: This comment should not be part of the commit.

File: utils/p2o.py [hunk 8]
```diff
@@ -34,6 +31,7 @@
 
 
 if __name__ == '__main__':
+    """Perceval2Ocean tool"""
```
Comment (sanacl) [inline utils/p2o.py]: This comment should not be part of the commit.

**Review by canasdiaz — CHANGES_REQUESTED** [review]
Some minor changes requested to make sure we don't break the aim of the commits according to the git log message, which is update the license info.

**Review by canasdiaz — APPROVED** [review]

**Pull request merged**